create.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles/index.css" />
+    <title>Create new question and answers</title>
+  </head>
+  <body>
+    <h1>Create questions and answers</h1>
+    <main>
+      <h2 id="add-question-heading”">Add new question and answer</h2>
+      <form aria-labelledby="”add-question-heading”" autocomplete="off">
+        <label for="add-question-field">Add Question: </label>
+        <input id="add-question-field" type="text" placeholder="Add Question here" required />
```

Comment:
really good structured 👍 just one thing...for the add question and add answer i would recommend `textarea`